lstmcpipe/workflow_management.py

```diff
@@ -9,12 +9,13 @@
 import pprint
 import calendar
 import lstchain
+from lstmcpipe.stages import (onsite_mc_merge_and_copy_dl1,
+                              onsite_mc_train,
+                              onsite_mc_dl2_to_irfs,
+                              onsite_mc_dl1_to_dl2
+                              )
```

Comment:
If you import the functions from the subscripts into `stages/__init__.py` and make them available via `__all__`, you could do `from lstmcpipe.stages import r0_to_dl1, dl1_to_dl2, ...` and directly use them avoiding the lengthy calls in the batch functions.

E.g. like here
https://github.com/cta-observatory/ctapipe/blob/master/ctapipe/core/__init__.py

Reply:
Thanks for the tips and the review ! Will commit it soon !